### ISSUE TITLE
How about support for posting keyboard and mouse events?

### ISSUE THREAD

**Issue by satyagraha** [kristian/system-hook#10]
Often we want to do the above when a specific kbd event happens...
Some clues: https://github.com/kwhat/jnativehook/blob/dd04fa987e1153cce078c859924f945307c04734/src/jni/org_jnativehook_GlobalScreen.c#L134

**Comment by kristian** [comment]
Good idea. Unfortunately I currently don't have the time to implement posting support. Nevertheless feel free to send me a pull request! Otherwise I unfortunalely would have to stall this. Also Linux support is still in my queue. Sorry. Issue stays open for reference, don't worry. :-)

**Comment by vpchelko** [comment]
Right now you can use Robot for posting events
